.github/workflows/submitty_ci.yml

```diff
@@ -231,32 +231,29 @@ jobs:
               sudo python3 -m pip install --upgrade pip setuptools wheel
               pip3 -V
               sudo pip3 -V
-              pip3 install cryptography
+              # Testing specific
+              pip3 install selenium
+              pip3 install psycopg2
```

Comment:
This is a central dependency of Submitty as it's the database driver we use anywhere where we use sqlalchemy.

Reply:
Understood